sdk/network/Microsoft.Azure.Management.Network/src/Generated/ConnectionMonitorsOperations.cs

```diff
@@ -831,10 +831,6 @@ public async Task<AzureOperationResponse<ConnectionMonitorResult>> BeginCreateOr
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "parameters");
             }
-            if (parameters != null)
-            {
-                parameters.Validate();
```

Comment:
Code in `Generated/` should not be modified manually. It should exactly match the spec.
Why did you do this?

Reply:
Ok...thanks.

I need to move the following three lines of codes from this API: how then can I remove it ? it is only in the \Generated\ folder?: Can you please how to remove these three lines of code from the specs?
            if (parameters != null) 
            { 
                parameters.Validate();
            }

Reply:
I'm not familiar with your spec but I'm sure it can be configured